### ISSUE TITLE
Just a few questions from a possibly user

### ISSUE THREAD

**Issue by Frontrider** [SpongePowered/Sponge#177]
I have only a few questions, based on what i have seen in the past.

- Bukkit was stable, until minecraft started to change significantly (namely 1.7), where bukkit has "broken minecraft". I'm not sure how you plan stability, at the point where there is no stability.

- If the mod api comes out for real, than forge possibly dies(like modloader did before, and i hope that it will, because that means that the api is good), because of the power of defaults.

-(coming from "why not wait for the official minecraft api?") We really need proper "official"(with this i mean something unlike a supportless bukkit/cauldron build) server side tools. Why not try to build something proper on top of forge for now, so people can use it (for now).

I have a few more, but no time to write, i will edit it later.

**Comment by ZephireNZ** [comment]
> Bukkit was stable, until minecraft started to change significantly (namely 1.7), where bukkit has "broken minecraft". I'm not sure how you plan stability, at the point where there is no stability.

Bukkit went from stable to unstable because of the major internal changes mojang made in 1.7 and 1.8. Sponge was conceived after that time, so is able to start from scratch taking those into account. There's always a change of major changes happening, only thing we can do is work around what happens.

> If the mod api comes out for real, than forge possibly dies(like modloader did before, and i hope that it will, because that means that the api is good), because of the power of defaults.

In all likeliness, the official API will not cover everything that Sponge, and certainly Forge, cover. That means there will still be a market for mods that make changes beyond what the API allows.

On top of that, Sponge API is designed from the ground up to support multiple implementations. The official one will be based upon Forge, however work is already underway to have Vanilla and Glowstone implementations.

**Comment by Frontrider** [comment]
I know how bukkit went from stable to unstable and i know the reasons because  felt it . I tried to use vanilla mechanics on a bukkit server,  and it wasn't working, because bukkit overridden too mutch. I hope that won't be your workaround to remove new things to keep the old ones working.(one example summon\ spawnmob,  summon wasn't at its full power,  while spawnmob wasn't proper).
Sometimes letting go is the answer.

I wanted to question why not forge directly,  but you answered that. 

I checked that you will do a q&a session, i will see if i can be there.
